Store descriptions in the scripts property

zkochan opened the pull request:
Add the possibility to store the descriptions not only in the `scripts-info` property but in the `scripts` property as well. Properties that are prefixed with ? in the `scripts` are descriptions for the corresponding scripts.

closes #6

zkochan committed:
feat(lib): store descriptions in the scripts property

Add the possibility to store the descriptions not only in the `scripts-info` property but in the `scripts` property as well. Properties that are prefixed with ? in the `scripts` are descriptions for the corresponding scripts.

closes #6

srph commented:
Thanks. Will get back to this PR after I take a nap

zkochan commented:
OK, cool

srph commented on lib/index.js:
```diff
@@ -27,4 +27,40 @@ function info() {
+
+  var woEchoDescription = description.replace('echo ', '');
+
+  return removeQuotes('"', removeQuotes("'", woEchoDescription));
```
I think you should use a trim function instead.

srph commented on lib/index.js:
```diff
@@ -27,4 +27,40 @@ function info() {
+  return scriptName[0] === '?';
+}
+
+function getScriptDecription(description) {
```
Typo, should be "De**s**cription". IMO, this is better named as `getDescription`

srph commented on lib/index.js:
```diff
@@ -27,4 +27,40 @@ function info() {
+  return scriptsInfo;
+}
+
+function scriptIsDescription(scriptName) {
```
imo, `isDescription` would be a better name.

zkochan committed:
feat(lib): store descriptions in the scripts property

Changes after CR. Use `unquote` for trimming quotes. Function name typo fix. Function names made shorter.

zkochan commented:
Thanks for checking the PR. I agree with all of your comments and I have updated the branch.

srph commented:
LGTM. Thanks a bunch! :100:
